src/Car.php

```diff
@@ -8,16 +8,16 @@ class Car
 
     private string $number_plate;
 
-    private float $fuelCosumption;
+
 
     public function __construct(
         int $odometer,
         string $number_plate,
-        float $fuelCosumption )
+        private float $fuelCosumption )
     {
         $this->odometer=$odometer;

```

Comment:
Instead of `$this->odometer=$odometer;`, more readable way it is `$this->odometer = $odometer;`

Based on PSR-12 standard - https://www.php-fig.org/psr/psr-12/#62-binary-operators

<img width="780" alt="image" src="https://user-images.githubusercontent.com/8157701/194484097-1780c3b7-5311-48da-adb7-b40e20827b01.png">